python/ray/serve/_private/logging_utils.py

```diff
@@ -123,8 +123,13 @@ def __init__(
         self,
         component_name: str,
         component_id: str,
+        fmt: Optional[str] = None,
+        datefmt: Optional[str] = None,
+        style: str = "%",
+        validate: bool = True,
     ):
-        self.component_log_fmt = ServeFormatter.COMPONENT_LOG_FMT.format(
+        super().__init__(fmt, datefmt, style, validate)
+        self.component_log_fmt = self.COMPONENT_LOG_FMT.format(
```

Comment:
nit: why is this now accessed with `self.COMPONENT_LOG_FMT`?

Reply:
its a mistake, changing it back.